noir/aztec_macros/src/lib.rs

```diff
@@ -1597,3 +1631,100 @@ fn event_signature(event: &StructType) -> String {
     let fields = vecmap(event.get_fields(&[]), |(_, typ)| signature_of_type(&typ));
     format!("{}({})", event.name.0.contents, fields.join(","))
 }
+
+fn aztec_prelude_source() -> &'static str {
+    let aztec_imports = "
+    use dep::aztec;
+
+    use dep::aztec::{
```

Comment:
Is there any downside to importhing things we're not using? (other than longer compilation time)

Reply:
If I have the `aztec` dependency and make a type of the same name of an import I will get a duplicate definition error. So anything imported in the prelude is essentially a reserved name.